contracts/actions/summerfi/SFApproveTokens.sol

```diff
@@ -0,0 +1,128 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.10;
+
+import {ActionBase} from '../ActionBase.sol';
+import {IERC20} from '../../interfaces/IERC20.sol';
+import {IDSProxy} from '../../interfaces/IDSProxy.sol';
+import {IExecutable} from '../../interfaces/summerfi/IExecutable.sol';
+import {IOperationsRegistry} from '../../interfaces/summerfi/IOperationsRegistry.sol';
+import {IOperationExecutor, Call} from '../../interfaces/summerfi/IOperationExecutor.sol';
+
+
+/// @title Approve tokens through Summerfi proxy
+contract SFApproveTokens is ActionBase {
```

Comment:
Would add comment that we need to permit address that will call through Summerfi proxy before calling this action

Reply:
legit